src/schema/rules/datatypes/microscopy.yaml

```diff
@@ -0,0 +1,46 @@
+---
+# Microscopy
+- suffixes:
+  - TEM
+  - SEM
+  - uCT
+  - BF
+  - DF
+  - PC
+  - DIC
+  - FLUO
+  - CONF
+  - PLI
+  - CARS
+  - 2PE
+  - MPE
+  - SR
+  - NLO
+  - OCT
+  - SPIM
+  extensions:
+  - .ome.tif
```

Comment:
Thank you @joshmoore for the feedback!

We were advised by BIDS to constrain to a single supported file extension in the case they all refer to the same file format. 

For regular Tiff, we went with `.ome.tif`.
For BigTiff, we understand that there is no difference between `.btf`, `.tf2` and `.tf8` based on information from this post (https://forum.image.sc/t/what-are-the-differences-between-ome-tif-ome-tiff-ome-tf2-ome-tf8-and-ome-btf-extensions/21304), and chose `.ome.btf`.

Please let us know if you foresee issues with this. Thank you!

Reply:
Only in the sense that some OME-TIFF files won't be BEP031-compatible but that's likely to be the case elsewhere as well (e.g. https://github.com/bids-standard/bids-specification/pull/881#discussion_r748404684)

Reply:
I think the idea of BIDS is to say: if you want to curate a Microscopy dataset in OME-TIFF format compatible with BIDS, use one of the supported extensions.

I assume file extensions can be changed "after the fact" while curating the dataset, but we can revisit this if that is not the case.

Reply:
Just to lay out the argument (as I suspect it's buried in the deep past of this BEP):

According to [[1](https://forum.image.sc/t/what-are-the-differences-between-ome-tif-ome-tiff-ome-tf2-ome-tf8-and-ome-btf-extensions/21304),[2](https://imagej.net/formats/bio-formats#bio-formats-exporter)], there is a meaningful distinction (at least for some tools) between `.ome.tif` and `.ome.btf` (TIFF and BigTIFF). But for the BigTIFF extensions, the file contents are the same.

While tool developers want to accept any valid filename, we are setting the conventions. Flexibility for a tool permits it to be used broadly, but flexibility in a file format or directory structure is only useful if it permits an additional use case. Multiple possible extensions for the same data don't do that.

Given that people will already need to rename files to start with `sub-`, they can just as easily rename them to end with `.ome.btf` instead of `.ome.tf2`. These will continue to be readable to tools that accept both, and researchers examining multiple BIDS datasets won't need to concern themselves with whether two extensions reflect the same or different file types.

Reply:
i believe the only difference is that a standard tiff reader can read `ome.tiff`, since the ome part is in the metadata of the file, so just like the `png` option there may be a utility reason to have `ome.tiff` as an option. i don't know if a standard tiff reader can read the `btf` named extension simply based on file magic.

Reply:
Thank you @effigies for the clarification!

@satra, you are right, "standard" tiff reader are usually able to read `.ome.tif` and `.ome.tiff`, but not `.ome.btf`, `.ome.tf2 `and `.ome.tf8` (in my experience.) But those BigTIff files can be read with ImageJ/Bioformats and tifffile for examples.

This we made the distinction by accepting 1 extension for each i.e. `.ome.tif` and `.ome.btf` (TIFF and BigTIFF).

Reply:
@satra bigtiff can be distinguished from tiff by the version # in bytes 2 & 3 of the header. tiff has a version # of 42 and bigtiff has a version # of 43.